constructor_io/constructor_io.py

```diff
@@ -165,6 +159,31 @@ def patch_batch(self, items, autocomplete_section, **kwargs):
             self._format_items_batch_data(items, autocomplete_section, **kwargs),
         )
 
+    def get_refined_queries(self, autocomplete_section, query=None):
+        args = {'autocomplete_section': autocomplete_section}
+        if query:
+            args['query'] = query
+        url = self._make_url("v1/refined_queries", args)
+        resp = self.__make_server_request(requests.get, url)
+        return resp.json()['refined_queries']
+
+    def put_automatically_generated_refined_queries(
+            self, refined_queries, autocomplete_section
+    ):
```

Comment:
Since we should not expose this to customers, this should probably not be in the python client.